addons/bitbucket/utils.py

```diff
@@ -9,9 +9,9 @@
 def get_path(kwargs, required=True):
     path = kwargs.get('path')
     if path:
-        return urllib.unquote_plus(path)
+        return unquote_plus(path)
```

Comment:
I see the importance/necessity  of having tests for these functions, because even a minor deviations could cause problems, but I don't think it's really efficient here because:

1. `urllib.unquote_plus` and `future.moves.urllib.parse.unquote_plus` are exactly not too versions of the same modules, they are exactly the same module, just moved to a place that's accessible to both 2 and 3

2. Even if I wrote tests for these functions (and they do need tests.) I wounldn't know what edge cases could potentially cause failures, I've tried stuff like `urlparse.urlparse('>+%%%%$%##%35%25<') == future_urlparse('>+%%%%$%##%35%25<')` and it works.

I do support raising coverage here, but in the interest of expedience we should put this off until later.

Reply:
I don't usually write new tests when I don't add or modify new functionality. There also the issue in this particular PR that many of these areas are flask related and should be just be removed, but that's outside the scope.

I'm also not crazy about testing external libraries at all, they have there own tests. I feel it's better to just be deliberate when updating, in this case particalarly these libraries are well-known and not being used in complex ways.